docs-2.0/4.deployment-and-installation/5.zone.md

```diff
@@ -1,43 +1,55 @@
-# Manage Zone
+# Manage Zones
+
+!!! enterpriseonly
+
+    This feature is only available in the NebulaGraph Enterprise Edition.
 
 The Zone is a logical rack of storage nodes in a NebulaGraph cluster. It divides multiple storage nodes into manageable logical areas to achieve resource isolation. At the same time, you can control the Graph service to access the replica data in the specified Zone to reduce traffic consumption and improve access efficiency. This article describes how to use the Zone feature.
 
 ## Principle
 
-Storage nodes can be added to a Zone. When creating a graph space, you can specify a list of Zones. The graph space is created and stored on the storage nodes added in these Zones. Partition replicas are evenly stored in the Zones, as shown in the following figure.
+Within NebulaGraph, you can set up multiple Zones, with each Zone containing one or more Storage nodes. When creating a graph space, the system automatically recognizes these set Zones and stores graph space data on the Storage nodes within these Zones.
+
+It's important to note that when creating a graph space, you need to specify the number of replicas for data partitioning. At this point, **the specified number of partition replicas and the number of set Zones must be equal, otherwise, the graph space cannot be created**. This is because NebulaGraph evenly stores each replica of the graph space data across these set Zones.
```

Comment:
Suggest change to be more precise:

`This is because NebulaGraph evenly distributes the replicas of a graph space partition across these set Zones.`